_episodes/08-adding-ci-to-your-code.md

```diff
@@ -72,6 +72,11 @@ The compilation will result in an output binary called `skim`.
 
 ## Check jobs
 
+>## `act` is optional.
+>
+> Remember that `act` is not required but encouraged for completeing this workshop.
```

Comment:
Typo `completeing`

Reply:
.. is not required but encouraged for local testing. Alternatively you can always push and see the output in the browser.